src/analytics/GoogleAnalytics.js

```diff
@@ -0,0 +1,65 @@
+function doNotTrackEnabled() {
+  //        Standard                     IE 9/10              IE11/MS Edge
+  return navigator.doNotTrack || navigator.msDoNotTrack || window.doNotTrack;
+}
+
+function isTrackingEnabled() {
+  const gaAvailable = window.ga !== null && window.ga !== undefined;
+
+  if (!gaAvailable) {
+    console.warn('Google Analytics is unavailable. Unable to send any events.');
+  }
+
+  return gaAvailable && !doNotTrackEnabled();
+}
+
+function sendEvent(event) {
+  console.log('Sending GA event: ', event);
+  window.ga('send', event);
+}
+
+function sendPageView() {
+  sendEvent('pageview');
+}
+
+function set(field, params) {
+  window.ga('set', field, params);
+}
+
+function setCurrentPage(page) {
+  set('page', page);
+}
+
+const GoogleAnalytics = () => {
+  const enabled = isTrackingEnabled();
+  return {
+    anonymizeIpAddress() {
+      if (enabled) {
+        set('anonymizeIp', true);
```

Comment:
I think we should always anonymize IPs.

Reply:
Done https://github.com/creativecommons/cccatalog-frontend/pull/190/commits/2a6a9cf16a9a2eeb3639402bf76032e14e75089e